src/Compilers/Core/Portable/Symbols/SymbolEqualityComparer.cs

```diff
@@ -16,16 +16,17 @@ public sealed class SymbolEqualityComparer : IEqualityComparer<ISymbol?>
         /// <summary>
         /// Compares two <see cref="ISymbol"/> instances based on the default comparison rules, equivalent to calling <see cref="IEquatable{ISymbol}.Equals(ISymbol)"/>
         /// </summary>
-        public static readonly SymbolEqualityComparer Default = new SymbolEqualityComparer(TypeCompareKind.AllNullableIgnoreOptions);
+        public static readonly SymbolEqualityComparer Default = new(TypeCompareKind.AllNullableIgnoreOptions);
 
         /// <summary>
         /// Compares  two <see cref="ISymbol"/> instances, considering their nullability
         /// </summary>
-        public static readonly SymbolEqualityComparer IncludeNullability = new SymbolEqualityComparer(TypeCompareKind.ConsiderEverything2); //TODO: should this be explicitly *not* compare everything
+        public static readonly SymbolEqualityComparer IncludeNullability = new(TypeCompareKind.ConsiderEverything2); //TODO: should this be explicitly *not* compare everything
 
         // Internal only comparisons:
-        internal static readonly SymbolEqualityComparer ConsiderEverything = new SymbolEqualityComparer(TypeCompareKind.ConsiderEverything);
```

Comment:
Please don't reformat existing code, we prefer to leave things as they are. It helps make `git blame` more useful by not having otherwise-unrelated PRs in the history.

Reply:
Ok, I have changed it back. Though I don't like much that this means that the code is probably never going to be updated. 🙂

Reply:
We prefer to have git blame be useful 😉.